incubator/binding-pgsql-kafka/pom.xml

```diff
@@ -42,8 +42,9 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
-      <groupId>com.github.jsqlparser</groupId>
-      <artifactId>jsqlparser</artifactId>
+      <groupId>io.aklivity.zilla</groupId>
+      <artifactId>binding-pgsql</artifactId>
+      <scope>provided</scope>
```

Comment:
This `provided` scope should be removed - similar to `openapi` dependency in `openapi-asyncapi` binding.